server/config_manager/config_manager.go

```diff
@@ -43,18 +45,29 @@ var (
 	errNotSupported = "not supported"
 )
 
+// Server is the interface for configuration manager.
```

Comment:
Does it seem that we have another interface called `Server`? Maybe we should define the interface with some specifications.

Reply:
Yes, and actually we indeed have some interfaces called Server in different packages. Maybe we can handle it in the future. But I don't have a good idea.